src/main/java/ch/njol/skript/effects/EffSprintServer.java

```diff
@@ -0,0 +1,84 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.bukkitutil.ServerUtils;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Timespan;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.util.Kleenean;
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Sprint Server")
+@Description({
+	"Requests the server to sprint for a certain amount of time, or stops the server from sprinting.",
+	"Sprinting is where the server increases the tick rate depending on the time you input, and resets the tick rate to what it was after the server has finished sprinting."
+})
+@Examples({
+	"request server to sprint for 10 seconds",
+	"make server stop sprinting"
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.20.4+")
+public class EffSprintServer extends Effect {
+
+	static {
+		if (ServerUtils.isServerTickManagerPresent())
+			Skript.registerEffect(EffSprintServer.class,
+				"request [for] [the] server [to] sprint for %timespan%",
```

Comment:
is there a chance that the "request" is denied? if not, we should support `make the server sprint`

Reply:
I'm not sure, i made it "request" because thats the name of the method, **request**GameToSprint()

Reply:
I can make "make" an option though, even though there could be a chance it fails, though on the javadocs it doesn't say **how** it could potentially fail, so maybe its poor method naming?